examples/web_agent/env.example

```diff
@@ -0,0 +1,6 @@
+# Copy this file to .env and set the appropriate environment variable(s) for your application.
+AIRTOP_API_KEY=""
```

Comment:
you should prompt the user for the key

Reply:
that's fine? unless im missing something here - they run the script, they input their API key, and then just leave the script to do its thing?